# from django.core.exceptions import ValidationError


class TourValidator:
    def __init__(self): ...

    def __call__(self):
        return True
